src/util/finite_field.h

```diff
@@ -0,0 +1,195 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Alex Ozdemir
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2022 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * A finite-field element, implemented as a wrapper around Integer.
+ *
+ * TODOs:
+ * * consider montgomery form
+ * * extend to non-prime fields
+ */
+
+#include "cvc5_public.h"
+
+#ifndef CVC5__FINITE_FIELD_H
+#define CVC5__FINITE_FIELD_H
+
+#include <iosfwd>
+#include <iostream>
```

Comment:
If we include `iostream` in the header, there is no need to include `iosfwd`. `iosfwd` just contains forward declarations for `iostream`. If possible, we should avoid including `iostream`, in the header, though, since it is big.

Reply:
:+1: